src/clu_main.c

```diff
@@ -40,7 +40,10 @@ static const struct option mode_options[] = {
     {"enc",       no_argument,       0, WOLFCLU_CRYPT     },
     {"bench",     no_argument,       0, WOLFCLU_BENCHMARK },
     {"hash",      required_argument, 0, WOLFCLU_HASH      },
-    {"md5",       no_argument,       0, WOLFCLU_MD5       },
+    {"md5",       no_argument,       0, WOLFCLU_ALG_HASH  },
+    {"sha256",    no_argument,       0, WOLFCLU_ALG_HASH  },
+    {"sha384",    no_argument,       0, WOLFCLU_ALG_HASH  },
+    {"sha512",    no_argument,       0, WOLFCLU_ALG_HASH  },
```

Comment:
This comment goes with the string compare comment, if we can distinguish the hash type when initially processing the command then we can avoid checking it again, please expand on WOLFCLU_MD5 and use WOLFCLU_SHA256 etc.

Reply:
Fixed.